src/overlay/test/OverlayTests.cpp

```diff
@@ -189,27 +181,20 @@ TEST_CASE("loopback peer flow control activation", "[overlay][flowcontrol]")
         SECTION("basic")
         {
             // Successfully enabled flow control
-            runTest({cfg1, cfg2}, Peer::FlowControlState::ENABLED, false);
+            runTest({cfg1, cfg2}, true, false);
         }
         SECTION("bad peer")
         {
             // Try to disable flow control after enabling
-            runTest({cfg1, cfg2}, Peer::FlowControlState::ENABLED, true);
+            runTest({cfg1, cfg2}, true, true);
         }
     }
     SECTION("one disables")
     {
         // Peer tries to disable flow control during auth
         // Set capacity to 0 so that the peer sends SEND_MORE with numMessages=0
         cfg2.PEER_FLOOD_READING_CAPACITY = 0;
-        runTest({cfg1, cfg2}, Peer::FlowControlState::DISABLED);
-    }
-    SECTION("one does not support")
```

Comment:
This test needs to fail with "wrong protocol version"

Reply:
I removed it since (If I'm not mistaken!) the original intent of this test is no longer relevant. This test was about a peer which doesn't support flow control but passes the min overlay version check. But such a case no longer exists as the min overlay version supports flow control. And we have a test called `"reject peers with incompatible overlay versions"` about the case when the peer's version doesn't meet the min overlay version requirement and this test indeed fails with `wrong protocol version`